packages/tags/src/tags.ts

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+    html,
+    LitElement,
+    CSSResultArray,
+    TemplateResult,
+    queryAssignedNodes,
+} from 'lit-element';
+import { FocusVisiblePolyfillMixin } from '@spectrum-web-components/shared/lib/focus-visible.js';
+
+import { Tag } from './tag.js';
+
+import styles from './tags.css.js';
+
+/**
+ * @element sp-tags
+ */
+export class Tags extends FocusVisiblePolyfillMixin(LitElement) {
+    public static get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @queryAssignedNodes('')
```

Comment:
This is new to me, definitely using this in future

Reply:
Pretty new. Only available in the newest LitElement version from a couple weeks back.